pallets/loans/src/tests/borrow_loan.rs

```diff
@@ -19,10 +21,7 @@ fn config_mocks(withdraw_amount: Balance) {
 	});
 	MockPrices::mock_register_id(|id, pool_id| {
 		assert_eq!(*pool_id, POOL_A);
-		match *id {
-			REGISTER_PRICE_ID => Ok(()),
-			_ => Err(PRICE_ID_NO_FOUND),
-		}
```

Comment:
Why this removal? I think this should work for the case of the price is not found, which means that you've correctly used the `latest_settlement_price` instead